packages/starknet-snap/src/utils/din.ts

```diff
@@ -0,0 +1,23 @@
+import { constants } from 'starknet';
```

Comment:
IMO, we may better re-name the file with rpc-provider rather than din, 
as i think for other network it will be just the same function

and having the secret in config.ts to align others